aiohttp/client.py

```diff
@@ -54,7 +54,7 @@ def __init__(self, *, connector=None, loop=None, cookies=None,
         if loop.get_debug():
             self._source_traceback = traceback.extract_stack(sys._getframe(1))
 
-        if not loop.is_running():
+        if not loop.is_running() and loop is not None:
```

Comment:
I fixed an issue in the None check I wrote ``is not none`` instead of ``is None``.